### PR TITLE
show category details

### PR DESCRIPTION
**Pull request by quanvhframgia**

### CODE REVIEW

File: resources/lang/en/category.php [hunk 1]
```diff
@@ -10,5 +10,6 @@
     'edit_category' => 'Edit Category',
     'category_name' => 'Category Name',
     'parent_category' => 'Parent Category',
-    'no_results' => 'No Results'
+    'no_results' => 'No Results',
+    'lessons_of_category' => 'List lessons of this category'
```
Comment (huynguyenframgia) [inline resources/lang/en/category.php]: Em thêm dấu `,` cuối nhé

File: resources/views/admin/category/show.blade.php [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+@section('content')
+    <div class="col-md-offset-1 col-md-10">
+        <div class="content-box-header">
+            <div class="panel-title">{{ trans('category.details') }}</div>
+        </div>
+        <div class="content-box-large box-with-header">
+            <div class="row">
+                <div class="col-md-5">
+                    <h3>{{ $category->name }}</h3>
+                    <p><strong>{{ trans('category.parent_category') }}</strong> {{ $category->parentcategory->name or '' }}</p>
```
Comment (huynguyenframgia) [inline resources/views/admin/category/show.blade.php]: parentCategory

File: app/Http/Controllers/Admin/CategoryController.php [hunk 3]
```diff
@@ -79,10 +79,17 @@ public function destroy($id)
...
+    
         return redirect()->back()->with([
             'status' => 'danger',
             'message' => trans('messages.admin.categories.delete.failed')
         ]);
     }
+    
+    public function show($id)
+    {
+        $category = $this->categoryRepository->find($id);
```
Comment (quanvhframgia) [inline app/Http/Controllers/Admin/CategoryController.php]: đã thêm

File: resources/lang/en/messages.php [hunk 4]
```diff
@@ -13,6 +13,9 @@
             'delete' => [
                 'success' => 'Delete category success!',
                 'failed' => 'Delete category failed!'
+            ],
+            'show' => [
+                'not_found' => 'This category does not exist!'
```
Comment (huynguyenframgia) [inline resources/lang/en/messages.php]: Giong https://github.com/framgia/fels_186/pull/15/files#diff-ea4531a0d6bda57d31796f8b0384432eR14

**Comment by huynguyenframgia** [comment]
Ok

**Pull request merged**